openmetadata-ui/src/main/resources/ui/src/components/Database/Profiler/TableProfiler/TableProfilerChart/TableProfilerChart.tsx

```diff
@@ -104,7 +104,8 @@ const TableProfilerChart = ({
   );
   const [operationDateMetrics, setOperationDateMetrics] =
     useState<MetricChartType>(INITIAL_OPERATION_METRIC_VALUE);
-  const [isLoading, setIsLoading] = useState(true);
+  const [isTableProfilerLoading, setIsTableProfilerLoading] = useState(true);
+  const [isSystemProfilerLoading, setIsSystemProfilerLoading] = useState(true);
```

Comment:
You can go for number based loading to keep one loader
setIsLoading(prev => prev+1)
setIsLoading(prev => prev-1)

isLoading > 0 ? Loader : render